products/quarkus.md

```diff
@@ -68,8 +68,8 @@ releases:
 
 -   releaseCycle: "2.7"
     eol: 2022-05-30
-    latest: "2.7.6"
-    latestReleaseDate: 2022-05-30
+    latest: "2.7.7"
```

Comment:
2.16 officially released yesterday (). I wonder if we should not consider Quarkus released when binaries are available on Maven Central (=> switch to Maven automation), because it is available and anyway release dates will be wrong.

Reply:
+1. Maven dates are always better.

Reply:
Note: did not switch to maven in #2391. I was not sure the regex would work.